lib/l2meter/emitter.rb

```diff
@@ -1,5 +1,4 @@
 require "time"
-require "thread"
```

Comment:
You don't need to require thread to get Mutex defined anymore? Is this a change in newer ruby versions?

Reply:
As of Ruby 2.2-ish? Anyhow, it's one of the "safe auto fixes" that `standard` does for us.